pl_pl.lang

```diff
@@ -108,104 +109,362 @@ gui.xaero_quick_use_6=Szybkie użycie 6
+gui.xaero_box_pushbox_potion_effects=Niewidzialny obszar otaczający miejsce z Minecrafta vanilla, gdzie wyświetlane są efekty mikstur, który przepycha interfejsy moda tak, aby nie kolidowały. \n \n Obszar wypychania typu normalnego jest równy jego rozmiarowi i przepycha interfejsy tylko o niezbędną odległość. \n \n Obszar wypychania typu wysokość ekranu rozciąga się na całej wysokości ekranu i zawsze przepycha pionowo kolidujące interfejsy o wysokość typu normalnego. Może to być bardzo przydatne, gdy masz wiele interfejsów w jednej kolumnie i chciał(a)byś, aby wszystkie z nich były przepychane.
+gui.xaero_held_centered_position=Przedm. po wyśrodkowaniu
+gui.xaero_held_centered_position_screen_edges=Krawędzie
+gui.xaero_held_centered_position_hotbar=Pasek
+gui.xaero_minimap_frame=Styl ramki
+gui.xaero_minimap_frame_color=Kolor ramki
+gui.xaero_box_minimap_frame_color=Kolor ramki minimapy dla kolorowych stylów.
+gui.xaero_minimap_frame_default=Domyślny
+gui.xaero_minimap_frame_colored_thick=Kolorowy
+gui.xaero_minimap_frame_colored_thin=Kolorowy cienki
+gui.xaero_compass_north=N
+gui.xaero_compass_east=E
+gui.xaero_compass_south=S
+gui.xaero_compass_west=W
+gui.xaero_compass_scale=Skala kompasu na mapie
+gui.xaero_box_compass_scale=Skala liter kompasu (N, E, S, W), gdy kompas znajduje się na mapie.
+gui.xaero_minimap_compass_inside_frame=Na mapie
+gui.xaero_minimap_compass_on_frame=Na ramce
+gui.xaero_compass_color=Kolor cienia kompasu
+gui.xaero_box_compass_over_wp2=Renderowanie liter kompasu (N, E, S, W) nad waypointami na mapie. Działa to tylko wtedy, gdy kompas znajduje się na ramce.
+effect.xaerominimap.no_minimap=Bez minimapy
+effect.xaerominimap.no_entity_radar=Bez radaru bytów
+effect.xaerominimap.no_waypoints=Bez waypointów
+effect.xaerominimap.no_cave_maps=Bez tryb. jaskiniow.
+effect.xaerominimap.no_minimap_harmful=Bez minimapy
+effect.xaerominimap.no_entity_radar_harmful=Bez radaru bytów
+effect.xaerominimap.no_waypoints_harmful=Bez waypointów
+effect.xaerominimap.no_cave_maps_harmful=Bez tryb. jaskiniow.
+effect.xaerominimap.no_minimap_beneficial=Bez minimapy
+effect.xaerominimap.no_entity_radar_beneficial=Bez radaru bytów
+effect.xaerominimap.no_waypoints_beneficial=Bez waypointów
+effect.xaerominimap.no_cave_maps_beneficial=Bez tryb. jaskiniow.
+gui.xaero_radar_item_frames=Wyświet. kropek ramek na przedm.
+gui.xaero_show_dimension_name=Wyświetlanie bieżącego wymiaru
+gui.xaero_unknown_biome=Unknown Biome
```

Comment:
Not translated!

Reply:
Used when a biome in the game world doesn't have an ID for some reason.

Reply:
Thank you for the context. This is now translated.